build.proj

```diff
@@ -41,11 +24,9 @@
     </Project>
     <!- signing must happen before packaging -!>
     <Project Include="src\sign.builds" />
-    -->
     <Project Include="src\packages.builds" Condition="'$(BuildPackages)'=='true'">
```

Comment:
That said we could probably keep it referenced here and just not have it build by default and people could still pass BuildPackages=true to get it included.